server/models/Posts.js

```diff
@@ -0,0 +1,14 @@
+const mongoose = require('mongoose');
+
+const PostSchema = new mongoose.Schema({
+    created   : {
+		type    : Date,
+		default : Date.now
+	},
+    content: String,
+    image: String
```

Comment:
We need to add a user and likes fields in order for service-functions to work. (later we will worry about connecting them)
 
like so: 

user: String,
likes: Boolean